ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -473,12 +474,19 @@ function TrackerScreen.drawScreen()
 
 	local viewedPokemon = Tracker.getPokemon(Tracker.Data.ownViewSlot, true)
 	local opposingPokemon = Tracker.getPokemon(Tracker.Data.otherViewSlot, false)
+	if Battle.isGhost and opposingPokemon ~= nil and opposingPokemon.level ~= nil then
+		--Ghost's level displays in battler, so keep that in the Ghost data
+		local prevOpposingPokemonLevel = opposingPokemon.level
+		opposingPokemon = Tracker.getDefaultPokemon()
+		opposingPokemon.level = prevOpposingPokemonLevel
+	end
 
 	-- Depending on which pokemon is being viewed, draw it using the other pokemon's info for calculations (effectiveness/weight)
 	if not Tracker.Data.isViewingOwn and opposingPokemon ~= nil then
 		local tempPokemon = viewedPokemon
 		viewedPokemon = opposingPokemon
 		opposingPokemon = tempPokemon
+		viewedPokemonLevel = viewedPokemon.level
```

Comment:
Looks like `viewedPokemonLevel` isn't defined or used anywhere else but here. I suspect we can remove this line of code?